tensorrt_llm/_torch/models/modeling_qwen3vl.py

```
@@ -624,17 +651,15 @@ def prepare_attn_metadata(self, seq_lens, attn_metadata: AttentionMetadata):
         return attn_metadata
 
     @torch.inference_mode()
-    def forward(
-        self, hidden_states: torch.Tensor, grid_thw: torch.Tensor, **kwargs
-    ) -> torch.Tensor:
+    def forward(self, pixel_values: torch.Tensor, grid_thw: torch.Tensor, **kwargs) -> torch.Tensor:
```

Ruff (0.14.10)

Unused method argument: `kwargs`

(ARG002)